src/App.scss

```diff
@@ -20,3 +20,65 @@ body {
     align-items: center;
   }
 }
+
+.SubNavbar {
+  background: transparent !important;
+  height: 42px;
+  font-family: "Roboto", sans-serif;
+  font-size: 13px;
+
+  ul.Menu {
```

Comment:
- remove tag from list selectors
- use kebab case for naming eg menu, menu-item